File: src/main/java/io/github/dqualizer/dqlang/types/openapiadapter/data/Property.java
package io.github.dqualizer.dqlang.types.openapiadapter.data;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Getter
@ToString
@AllArgsConstructor
@NoArgsConstructor
public class Property {

    private String name;
    private String type;
}

File: src/main/kotlin/io/github/dqualizer/dqlang/archive/openapiadapter/dqlang/data/Property.kt
package io.github.dqualizer.dqlang.archive.openapiadapter.dqlang.data

import lombok.AllArgsConstructor
import lombok.Getter
import lombok.NoArgsConstructor
import lombok.ToString

@Getter
@ToString
@AllArgsConstructor
@NoArgsConstructor
class Property {
    val name: String? = null
    val type: String? = null
}
